### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by NDBman**

### CODE REVIEW

File: groupnews-web/src/main/resources/logback-test.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (uzonyib) [inline groupnews-web/src/main/resources/logback-test.xml]: Why is this config for test context? It should be simply logback.xml to also ensure logging when you start the Boot app.

**Review by uzonyib — CHANGES_REQUESTED** [review]

File: groupnews-web/src/main/resources/logback-test.xml (hunk 1 above)
Comment (NDBman) [reply groupnews-web/src/main/resources/logback-test.xml]: At logback's webpage in charter configuration it was written that firstly it tries to search for logback-test.xml so I guess I just created that file without think about the logic of it. Will correct it.

Comment (NDBman) [reply groupnews-web/src/main/resources/logback-test.xml]: Chapter*

File: groupnews-web/src/main/java/com/epam/internship/controller/UserController.java [hunk 2]
```diff
@@ -18,12 +20,15 @@
 @RestController
 @RequestMapping("/")
 public class UserController {
+	
+	final static Logger logger = LoggerFactory.getLogger(UserController.class);
```
Comment (uzonyib) [inline groupnews-web/src/main/java/com/epam/internship/controller/UserController.java]: private static final Logger LOG(GER)

**Review by uzonyib — APPROVED** [review]

**Review by Attila-Neuwirth — APPROVED** [review]

**Pull request merged**